Portfolio/index.html

```diff
@@ -13,18 +14,22 @@
     <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
     <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
     <nav>
+    <h2>Jahnavi</h2>
         <ul>
             <li>Home</li>
             <li>About</li>
             <li>Skills</li>
             <li>Contact</li>
         </ul>
     </nav>
-    <h1>Hello I'm</h1>
-    <h2>Jahnavi</h2>
-    <img src="https://png.pngtree.com/element_our/20190601/ourlarge/pngtree-desktop-computer-one-flat-style-work-girl-vector-image_1358068.jpg" alt="working"/>
+    <i class="fa fa-phone"></i> <br>

```

Comment:
Consider adding content or structural elements within this empty line for better visual separation of sections in the HTML document.